pkg/tasks/monitoring_plugin.go

```diff
@@ -71,17 +71,6 @@ func (t *MonitoringPluginTask) Run(ctx context.Context) error {
 		}
 	}
 
-	{ // config map
-		cm, err := t.factory.MonitoringPluginConfigMap()
-		if err != nil {
-			return fmt.Errorf("initializing Console Plugin ConfigMap failed: %w", err)
-		}
-
-		if err = t.client.CreateOrUpdateConfigMap(ctx, cm); err != nil {
```

Comment:
we would need to delete the configmap (4.18) then remove the deletion in 4.19.

Reply:
I updated the PR to hardcode deleting the expected config map in the most recent commit